### DISCUSSION TITLE
在archives页面为分类添加的图片无法加载~

### DISCUSSION THREAD

**Discussion by sxfinn** [CaiJimmy/hugo-theme-stack#752] [Q&A]
在categories页面是能看到每个类别的图片的，但是archives页面就不行，我该如何解决呢？
问题如下图：
![image](https://user-images.githubusercontent.com/89447276/210612806-5f6f3fe2-e974-47a3-b880-059ae1ca08cd.png)
![image](https://user-images.githubusercontent.com/89447276/210612853-b64dac13-1b2f-4bd9-aadd-88dfdaaa9868.png)
我所有的更改都是并没有修改过主题的源码，我的为categories添加图片的布局如下：
![image](https://user-images.githubusercontent.com/89447276/210613171-bd5a55f2-5d78-4873-a8c2-3502a9e79bae.png)
![image](https://user-images.githubusercontent.com/89447276/210613228-11276b23-681b-4a00-9ba5-2b2084dddf6d.png)
并且我在本地预览时没有这个问题，一旦推送到GitHub page上就会有此问题。

我的博客链接：https://stack.xinsong.xyz

**Comment by CaiJimmy** [comment]
GitHub Page 的问题：https://github.com/CaiJimmy/hugo-theme-stack/issues/198#issuecomment-826964724
